mixer/pkg/protobuf/yaml/dynamic/handler.go

```diff
@@ -169,14 +172,27 @@ func (h *Handler) handleRemote(ctx context.Context, qr proto.Marshaler,
 		return err
 	}
 
-	if err := h.conn.Invoke(ctx, svc.GrpcPath(), ba, resultPtr); err != nil {
+	codec := grpc.CallCustomCodec(&Codec{decoder: svc.decoder})
```

Comment:
is it not possible to create codec in `connect` and then the connection will use it during invoke ?